packages/account/src/Sections/Verification/ProofOfIdentity/proof-of-identity-container.jsx

```diff
@@ -150,9 +152,9 @@ const ProofOfIdentityContainer = ({
 
     // component didUpdate hook, checks previous account_status and current account_status to handle account switching
     React.useEffect(() => {
-        if (account_status && previous_account_status) {
-            if (previous_account_status !== account_status) {
-                createVerificationConfig(account_status);
+        if (current_account_status && previous_account_status) {
```

Comment:
this is seems like a redundant variable, since we have account status, you may check use-previous, i think it behave the same way,

Reply:
What do you mean?
To make this clear, `previous_account_status` is the **previous** instance of `account_status` which is a prop. `current_account_status` is the **current** instance retrieved from the response in the componentDidMount hook.